### PR TITLE
DOC: Mention that we provide Python 3.10 wheels

### LINKED ISSUE #42946: Python 3.10 manylinux wheels
Hello.

Could you please release CPython 3.10 manylinux wheels on PyPI? 

Python 3.10.0 release candidate 1 was released this week: https://www.python.org/downloads/release/python-3100rc1/

It should be quite safe to release wheels for release candidate. ABI for Python 3.10 should be frozen from now on, even for bugfixes. From release notes: 

> There will be no ABI changes from this point forward in the 3.10 series and the goal is that there will be as few code changes as possible.

Thank You.

### PR DESCRIPTION
**Pull request by lithomas1**
- [ ] closes #42946
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

IMPORTANT: This depends on MacPython/pandas-wheels#152, so please merge that first.

### CODE REVIEW

**Comment by simonjayhawkins** [comment]
> IMPORTANT: This depends on [MacPython/pandas-wheels#152](https://github.com/MacPython/pandas-wheels/pull/152), so please merge that first.

Thanks @lithomas1 marked this one as draft for now.

**Review by simonjayhawkins — APPROVED** [review]

File: doc/source/whatsnew/v1.3.3.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ Bug fixes
 
 Other
 ~~~~~
--
+- Wheels are now provided for Python 3.10rc1 (:issue:`42946`)
```
Comment (alimcmaster1) [inline doc/source/whatsnew/v1.3.3.rst:37]: Do we want to actually provide wheels for the rc? Release is expected 4th Oct so maybe we just wait until then for this: https://www.python.org/dev/peps/pep-0619/#schedule

**Comment by jreback** [comment]
no we can build wheels for rc as a test
but don't add any notes until actual python release

**Comment by simonjayhawkins** [comment]
> but don't add any notes until actual python release

sure. will close this.

File: doc/source/whatsnew/v1.3.3.rst (hunk 1 above)
Comment (hroncok) [reply doc/source/whatsnew/v1.3.3.rst:37]: Yes, please provide wheels for the rc. See https://github.com/numpy/numpy/issues/19630#issuecomment-898314636

**Comment by hroncok** [comment]
> no we can build wheels for rc as a test
> but don't add any notes until actual python release

Could you please provide wheels built with rc? See https://github.com/numpy/numpy/issues/19630#issuecomment-898314636

**Comment by simonjayhawkins** [comment]
> Could you please provide wheels built with rc? See [numpy/numpy#19630 (comment)](https://github.com/numpy/numpy/issues/19630#issuecomment-898314636)

Thanks @hroncok for the question.

This issue is closed since we are not adding a release note about the availabilty of 3.10 wheels. 

see https://github.com/pandas-dev/pandas/issues/43050#issuecomment-912455816 for futher discussion on providing wheels.

**Pull request closed without merging**